src/Tree/index.tsx

```diff
@@ -50,6 +50,7 @@ class Tree extends React.Component<TreeProps, TreeState> {
     leafNodeClassName: '',
     renderCustomNodeElement: undefined,
     enableLegacyTransitions: false,
+    interactiveNode: false,
```

Comment:
See comment on `src/Tree/types.ts`

```suggestion
    hasInteractiveNodes: false,
```